oneanddone/functional_tests/requirements.txt

```diff
@@ -0,0 +1,4 @@
+bidpom
+pytest-selenium
+requests==2.8.1
+-e git+https://github.com/bobsilverberg/pytest-django.git@6d8d45c4fd1eb58b553045dd844e35885fcf67e6#egg=pytest_django
```

Comment:
I don't know why nobody else has run into this either. I opened the issue and if they fix it or suggest a resolution I will update the app accordingly.